private/bufpkg/bufmodule/bufmoduleref/bufmoduleref.go

```diff
@@ -28,9 +28,13 @@ import (
 )
 
 const (
+	// MainTrack is the name of the track created for every repository.
+	// This is the default track used if no track is specified.
+	MainTrack = "main"
+
 	// MainBranch is the name of the branch created for every repository.
 	// This is the default branch used if no branch or commit is specified.
-	MainBranch = "main"
+	MainBranch = MainTrack
```

Comment:
Separate from this PR, but why do we still need `MainBranch`?

Reply:
There is some dead branch code that uses it both in this repository and core. MainBranch won't be around for much longer.